lldb/source/Core/Progress.cpp

```diff
@@ -30,10 +30,9 @@ Progress::~Progress() {
   // Make sure to always report progress completed when this object is
   // destructed so it indicates the progress dialog/activity should go away.
   std::lock_guard<std::mutex> guard(m_mutex);
-  if (!m_completed) {
+  if (!m_completed)
     m_completed = m_total;
-    ReportProgress();
-  }
+  ReportProgress();
```

Comment:
Shouldn't this go upstream?

Reply:
@chelcassanova This should go upstream, before you land this patch otherwise, you could get a merge conflict.